### PR TITLE
feat(tag): added tag component

### PR DESCRIPTION
**Pull request by GiovannaMonti**
### Description

Added **Tag component** remapping only the fundamental props from the antd5 Tag.

### Checklist

- [x] commit message and branch name follow conventions
- [x] tests are included
- [x] changes are accessible and documented from components stories
- [ ] typings are updated or integrated accordingly with your changes
- [x] all added components are exported from index file (if necessary)
- [x] all added files include Apache 2.0 license
- [x] you are not committing extraneous files or sensitive data
- [x] the browser console does not have any logged errors
- [x] necessary labels have been applied to this pull request (enhancement, bug, ecc.)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ You can see the `npm` scripts available inside the `package.json` file and use t
 :warning: All changes **MUST** pass through merge request approvals.
 
 ## Before submitting a PR
-
+ * Your PR title should follow the `<type>(<scope>): <subject>` format, so that it gets correctly categorized in the release notes.
```
Comment (GiovannaMonti) [inline CONTRIBUTING.md:32]:
```suggestion
```

File: src/components/Tag/Tag.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  } : TagProps
+) : ReactElement => {
+  return (
+    <AntTag
+      bordered={isBordered}
+      closeIcon={closeIcon}
+      color={color}
+      onClose={onClose}
+    >
+      {children}
```
Comment (fredmaggiowski) [inline src/components/Tag/Tag.tsx:44]: I guess we are going to use it most of the time with our `FormattedMessage` component but a full ReactNode in the tag could be abused 🤔  would it make sense to enforce a Typography element or to enforce a prop definition compatbile with our typography?

Comment (Danielecina) [reply src/components/Tag/Tag.tsx:44]: I think it is important to ask the designer to know if the tag component will have in the future a specific size. But for i18n I suggest using a children's node

**Review by Danielecina — APPROVED** [review]

**Review by fredmaggiowski — APPROVED** [review]
LGTM

**Pull request merged**